packages/react-core/src/components/CalendarMonth/CalendarMonth.tsx

```diff
@@ -340,19 +322,11 @@ export const CalendarMonth = ({
                 type="number"
                 value={yearFormatted}
                 onChange={(ev: React.FormEvent<HTMLInputElement>, year: string) => {
-                  const newDate = new Date(focusedDate);
-                  if (dateIsFebruary29th(newDate) && !yearHasFebruary29th(+year)) {
-                    newDate.setDate(28);
-                    newDate.setMonth(1);
-                  }
-                  if (dateIsFebruary29th(initialDate) && yearHasFebruary29th(+year)) {
-                    newDate.setFullYear(+year);
-                    newDate.setDate(29);
-                  }
-                  newDate.setFullYear(+year);
+                  const newDate = changeYear(Number(year));
                   setFocusedDate(newDate);
                   setHoveredDate(newDate);
                   setShouldFocus(false);
+                  focusRef.current.blur(); // will unfocus a date when changing year via up/down arrows
```

Comment:
This shouldn't be necessary. Clicking the input arrows for the year will place focus on the input itself

Reply:
It looks like google calendar does not focus a date as scrolling through months.  Adam had pinged me about htis yesterday and the PF 5.0 version did nit focus a specific month.